dart/test/sentry_options_test.dart

```diff
@@ -37,8 +37,7 @@ void main() {
     final options = defaultTestOptions();
     // ignore: deprecated_member_use_from_same_package
     expect(options.logger, noOpLogger);
-    // ignore: deprecated_member_use_from_same_package
```

Comment:
We can also remove the `// ignore: deprecated_member_use_from_same_package` directives.